explorer/src/components/account/UpgradeableLoaderAccountSection.tsx

```diff
@@ -179,7 +180,14 @@ export function UpgradeableProgramDataSection({
         {account.details?.space !== undefined && (
           <tr>
             <td>Data (Bytes)</td>
-            <td className="text-lg-end">{account.details.space}</td>
+            <td className="text-lg-end">
+              <Downloadable
+                data={programData.data[0]}
+                filename={`bytecode-${account.pubkey.toString()}.txt`}
```

Comment:
another nit: Could you change the extension to `.bin` to indicate that the file is binary data now? I sort of prefer to drop the "bytecode-" prefix as well but not a big deal, choose whatever you think is clearest.

Reply:
Extension changed and prefix dropped, it did not bring much.